src/ovp_pipeline/commands/ui_server.py

```
@@ -1853,6 +1854,17 @@ def _spawn_action_worker_process(vault_dir: Path | str, *, interval_seconds: flo
 
 
 def _prewarm_ui_caches(vault_dir: Path | str) -> None:
+    """Warm the evolution-browser payload cache.
+
+    On the live operator vault this single call costs ~4 minutes
+    (``_compute_evolution_candidates`` reads every open-contradiction
+    claim's source markdown to extract dates, in an N×M nested loop).
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)